Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py

```diff
@@ -2022,6 +2022,7 @@ def get_entity(eiq_api):
 
     elif query_result is False:
         return "No entities found in EclecticIQ Intelligence Center."
+    return None
```

Comment:
Lets return a readable output message (rather than None) in case the query result does not fall into none of the cases.

Reply:
but it will change the flow of our existing customer

Reply:
Previously, when none of the cases were satisfied, nothing would be returned. If existing flows do not expect a return in such scenarios, introducing a new, readable output should be harmless.

This aligns with our standards for ensuring robust and predictable behavior.

Let me know if you want to discuss it.

Reply:
if a function does not return anything, by default return value will be None

Reply:
We can also do something like this which allows maintaining current flows in addition to complying with our code standards.
```suggestion
    return_results("No entities found in EclecticIQ Intelligence Center.")
    return None
```

Reply:
updated